### ISSUE TITLE
Feat: Provide a delete Review Option

### ISSUE THREAD

**Issue by Laxelspal** [singhJasvinder101/MyBlog#16]
**Description of the issue**
I want to provide a delete review option in the review , This delete review option will be only accessible to the users  who have created that review, otherwise It should not be visible. Whenever they click on this delete review option this review should be deleted.

Record
 I agree to follow this project's Code of Conduct
 I'm a GSSOC'24 contributor
 I want to work on this issue

Hi @singhJasvinder101 , please assign me this issue.

**Comment by singhJasvinder101** [comment]
Issue assigned to you.

**Comment by Laxelspal** [comment]
Can you please add the level label as well?

**Comment by singhJasvinder101** [comment]
Are you working on this issue ?

**Comment by Laxelspal** [comment]
Yes, but there is a problem occurring while running it on localhost. What to do?

Uploading Tech Blog - Google Chrome 2024-05-29 20-40-32.mp4…

**Comment by Laxelspal** [comment]
Yes, but there is a problem occurring while running it on localhost. What to do?


https://github.com/singhJasvinder101/MyBlog/assets/136887514/7575c197-a3f3-436f-954f-d9be2f1a2b5b

**Comment by Laxelspal** [comment]
Yes, but there is a problem occurring while running it on localhost. The page is loading, but it is not displaying correctly.



https://github.com/singhJasvinder101/MyBlog/assets/136887514/f166c4a0-c6f6-4dc3-b342-8ca17b30eb5e

**Comment by Sid-80** [comment]
@singhJasvinder101 I am willing to work on this issue!

**Comment by Sid-80** [comment]
@singhJasvinder101 I am not able to find the .env for client and server. Please help me!

**Comment by singhJasvinder101** [comment]
@Sid-80 you can find the env variables in .env.example file checkout
